core/frontend/src/tile/RealityTileTree.ts

```diff
@@ -10,6 +10,7 @@ import { assert, BeTimePoint } from "@bentley/bentleyjs-core";
 import { Transform } from "@bentley/geometry-core";
 import { ColorDef, Frustum, FrustumPlanes, ViewFlagOverrides } from "@bentley/imodeljs-common";
 import { IModelApp } from "../IModelApp";
+import { GraphicsCollectorDrawArgs } from "../imodeljs-frontend";

```

Comment:
Don't import from imodeljs-frontend from within the imodeljs-frontend package.

Reply:
Fixed.